src/transformers/models/owlvit/feature_extraction_owlvit.py

```diff
@@ -0,0 +1,228 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Feature extractor class for OwlViT."""
+
+from typing import List, Optional, Union
+
+import numpy as np
+from PIL import Image
+
+from ...feature_extraction_utils import BatchFeature, FeatureExtractionMixin
+from ...image_utils import ImageFeatureExtractionMixin, is_torch_tensor
+from ...utils import TensorType, is_torch_available, logging
+
+
+if is_torch_available():
+    import torch
+    from torch import nn
+
+logger = logging.get_logger(__name__)
+
+
+def center_to_corners_format(x):
```

Comment:
I see that the feature extractor is imported under an is_vision_available, meaning it only requires Pillow.

However, for postprocessing, it seems PyTorch is required. This is similar to DETR's feature extractor, and this also makes me wonder whether postprocessing should be part of the model, rather than the feature extractor.

Alternatively, it is possible that this postprocessing can happen using NumPy alone. However, this might become a bottleneck in case postprocessing has a big speedup on GPU (when computing metrics like mAP for instance)

cc @amyeroberts